src/pages/Thanks/ThanksPage.vue

```diff
@@ -387,6 +397,12 @@ export default {
 		const transactionId = this.$route.query?.kiva_transaction_id
 			? numeral(this.$route.query?.kiva_transaction_id).value()
 			: null;
+
+		// Check if transactionId is null, exit if missing
+		if (!transactionId) {
+			return false;
```

Comment:
What happens when in this case? What does the user end up seeing?

Reply:
Continuing beyond this point without a transaction id causes a readQuery error, similar to the prefetch error. Visitors would see the same message to login.

Reply:
From my read, nothing below this can function properly without successful query which will fail without the transaction id.